src/event/events.rs

```diff
@@ -210,8 +210,13 @@ impl<'a> Iterator for Iter<'a> {
 
 impl fmt::Debug for Events {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("Events")
-            .field("capacity", &self.capacity())
-            .finish()
+        let mut d = f.debug_struct("Events");
+        d.field("capacity", &self.capacity());
+
+        for event in self.iter() {
+            d.field("event", event);
```

Comment:
Now we get a bunch of "event" which seems odd. Not sure about a better alternative though.

Reply:
Removed field name "event", was printed twice.